### PR TITLE
test: Merge Import Token flow methods and ids in just one folder and files

### PR DESCRIPTION
**Pull request by SamuelSalas**
## **Description**

We are aiming to refactor the page objects in the modal folder so that they strictly follow the page object model pattern. This would aide in providing more readable and help standardize the way we create our tests. Because of the amount of files remaining, this issue will focus on working on three files to refactor, as well as their respective testIDS.

While doing some analysis we see that the ImportTokens and AddCustomTokenView classes could be merge together. So the refactoring of the test and classes should be implemented on both files.

NOTE: You should move the AddCustomTokenView and ImportTokens files into a token-import subfolder within the wallet folder for better organization and clarity.

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

Regressin test runs: https://app.bitrise.io/app/be69d4368ee7e86d/pipelines/ca857ba6-cebc-4903-8918-3d7dc3a51a36

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/pages/ImportTokenFlow/ConfirmAddAsset.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import {
```
Comment (cortisiko) [inline e2e/pages/ImportTokenFlow/ConfirmAddAsset.js:1]: Rename the folder for these two pages to IimportTokens` and potentially move it into the wallet folder. Do the same with the ImportNFT file as well: Place it into an ImportNFTs directory

**Review by cortisiko — CHANGES_REQUESTED** [review]
Small feedback, otherwise looks good

**Review by cortisiko — APPROVED** [review]
🌮 🌮

**Review by cortisiko — DISMISSED** [review]

**Review by Daniel-Cross — DISMISSED** [review]
Really nice use of constants for these testId's. I like it. One small fix to address but I'll approve.

File: app/components/UI/AssetList/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import StyledButton from '../StyledButton'; // eslint-disable-line  import/no-un
 import AssetIcon from '../AssetIcon';
 import { fontStyles } from '../../../styles/common';
 import Text from '../../Base/Text';
-import { ImportTokenViewSelectorsIDs } from '../../../../e2e/selectors/ImportTokenView.selectors';
+import { ImportTokenViewSelectorsIDs } from '../../../../e2e/selectors/wallet/ImportTokenView.selectors';
```
[the remark this answers is not in the document]
Comment (SamuelSalas) [reply app/components/UI/AssetList/index.js]: Great catch, just remove it

**Review by cortisiko — APPROVED** [review]

**Review by gambinish — APPROVED** [review]
Code looks good. Approving, but when I was trying to run CI locally, I was getting a failed build:

```
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/nicholasgambino/code/metamask-mobile/ios/build/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/RNDateTimePicker.build/Objects-normal/x86_64/RNDateTimePickerShadowView.o /Users/nicholasgambino/code/metamask-mobile/node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RNDateTimePicker' from project 'Pods')
(1 failure)
error Command failed with exit code 65.
```

Steps I took:

```
git pull
yarn setup:e2e
yarn start:ios:e2e
```

Approving now bc this feels unrelated, and CI tests are passing in Github. Will reach out in Slack for help.

**Pull request merged**